### ISSUE TITLE
When adding a checklist custom metadata field under Edits, it displays as a dropdown

### ISSUE THREAD

**Issue by jason-mightynice** [cgwire/kitsu#928]
Adding a checklist metadata field under the Edits context displays as shown in screenshot below.

checklists display as expected under Shots and Assets.

<img width="600" alt="Screen Shot 2022-08-15 at 5 57 05 pm" src="https://user-images.githubusercontent.com/109259714/184597916-b2612e80-b05b-4b0d-8bef-3d981ed20ad3.png">

Studio name:  Mighty Nice
Kitsu version: 0.14.18

**Comment by frankrousseau** [comment]
It should be fixed. Can you try again?

**Comment by jason-mightynice** [comment]
I tried in 0.15.8 with an existing test project. I got the same result. This is with a new Edit.

Create checkbox

<img width="591" alt="Screen Shot 2022-09-16 at 8 21 37 pm" src="https://user-images.githubusercontent.com/109259714/190622371-c05727b6-8332-4d78-a28b-9807c6df4d2e.png">

Add field

![image](https://user-images.githubusercontent.com/109259714/190622483-8b3d80ed-b438-4b3d-aad9-785644c64eb0.png)
